parent: !include default.yaml
# parent: !include robots/boxy.yaml
robot_interface:
  base:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/base/follow_joint_trajectory
    state_topic: /whole_body_controller/base/state
    robot_name: boxy
  torso:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/torso/follow_joint_trajectory
    state_topic: /whole_body_controller/torso/state
    robot_name: boxy
  neck:
    plugin: SendFollowJointTrajectory
    namespace: /scaled_pos_traj_controller/follow_joint_trajectory
    state_topic: /scaled_pos_traj_controller/state
    robot_name: boxy
  left_arm:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/left_arm/follow_joint_trajectory
    state_topic: /whole_body_controller/left_arm/state
    robot_name: boxy
  right_arm:
    plugin: SendFollowJointTrajectory
    namespace: /whole_body_controller/right_arm/follow_joint_trajectory
    state_topic: /whole_body_controller/right_arm/state
    robot_name: boxy